bakery/checkers/time.go

```diff
@@ -30,3 +32,38 @@ var TimeBefore = CheckerFunc{
 func TimeBeforeCaveat(t time.Time) Caveat {
 	return firstParty(CondTimeBefore, t.UTC().Format(time.RFC3339Nano))
 }
+
+// ExpireTime finds any caveats created by TimeBeforeCaveat and
```

Comment:
// ExpiryTime returns the minimum time of any time-before caveats
// found in the given slice and whether there were
// any such caveats found.

?